test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -279,6 +274,31 @@ def convert_date_time(cls, dt=None, fmt=None, syslog=False):
         else:
             return int(tm)
 
+    def get_syslog_date_str(self, logline=None):
+        if logline is None:
+            return None
+
+        if "T" in logline:
+            return logline[:19]
+        else:
+            return logline[:15]
+
+    def convert_syslog_time(self, dt=None):
+        if dt is None:
+            return None
+
+        if 'T' in dt:
+            dt_str = dt[:10] + " " + dt[12:19]
+            t = datetime.datetime.strptime(dt_str, "%Y-%m-%d %H:%M:%S")
+            return t
+        else:
+            t = datetime.datetime.strptime(dt, "%b %d %H:%M:%S")
```

Comment:
again, it's not obvious what this block of code is doing, please add a comment to explain